x/feemarket/spec/README.md

```diff
@@ -0,0 +1,24 @@
+<!--
+order: 0
+title: Feemarket Overview
+parent:
+  title: "feemarket"
+-->
+
+# Feemarket
+
+## Abstract
+
+This document specifies the feemarket module which allows to define a global transaction fee for the network.
```

Comment:
I think this could be completed and filled with more context, use case, implementation reference (EIP1559), etc.